docs/tutorial.rst

```diff
@@ -1766,10 +1769,10 @@ added into the media along with mannitol then the results might appear as follow
 
 .. code-block:: shell
 
-    EX_glc_e	-10.0	-2.0	|D-Glucose[C_e]| <=>
-    EX_manni_e	-9.0	-3.0	|Mannitol[C_e]| <=>
-    MANNIPTS	3.0	9.0	|Mannitol[C_e]| + |Phosphoenolpyruvate[C_c]| => |Mannitol 1-phosphate[C_c]| + |Pyruvate[C_c]|
-    GLCpts	2.0	10.0	|D-Glucose[C_e]| + |Phosphoenolpyruvate[C_c]| => |Pyruvate[C_c]| + |D-Glucose-6-phosphate[C_c]|
+    EX_glc_e	-10.0	-2.0	|D-Glucose[e]| <=>
+    EX_manni_C_e	-9.0	-3.0	|Mannitol[e]| <=>
```

Comment:
your comment about the output not being correct doesn't show me the output you're talking about in the link?